File: lectures/code/cc_02_09.py
while True:
    try:
        line = input()
    except:  # If we get EOF
        break
    line = line.strip()
    guess = int(line)
    if guess == 42:
        print('Nice work!')
        break
    elif guess < 42 : 
        print('Too low - guess again')
    else :
        print('Too high - guess again')

File: lectures/code/cc_02_09.c
#include <stdio.h>
int main() {
    int guess;
    while(scanf("%d",&guess) != EOF ) {
        if ( guess == 42 ) {
            printf("Nice work!\n");
            break;
        }
        else if ( guess < 42 ) 
            printf("Too low - guess again\n");
        else
            printf("Too high - guess again\n");
    }
}
